### ISSUE TITLE
Post to tastejs/todomvc

### ISSUE THREAD

**Issue by tipiirai** [txchen/feplay#4]
I think this Riot 2.0 example should be posted to the official Todo MVC repository. They are not too open to new suggestions so I suggest to read the [contributing](https://github.com/tastejs/todomvc/blob/master/contributing.md) document very closely. 

Here is a very interesting discussion why we were rejected [last time](https://github.com/tastejs/todomvc/pull/744).

Willing to take a shot?

**Comment by txchen** [comment]
@tipiirai Yeah, I'd like to have a try. 

However before that, can I ask you to do a code review? Basically I'd like to ensure my code is following a correct or suggested style of writing riotjs.

**Comment by txchen** [comment]
@tipiirai FYI, I have started this task.
https://github.com/txchen/todomvc/commit/0e4a6aea06c337a7cb38faa8136d5716d692ecf5

Currently the browser tests are not passing, I will investigate. Once it is ready, I will send the PR.

In the mean while, would you please have a review on this?

**Comment by txchen** [comment]
Blocked by browser test.
Please take a look at this and help, thanks! https://github.com/muut/riotjs/issues/413

**Comment by txchen** [comment]
It is merged into todomvc.

**Comment by tipiirai** [comment]
Awesome.

Thank you for the hard push on this! Looks like we're under the "labs" section on the home page: http://todomvc.com/
